### PR TITLE
add docs of static method

### PR DESCRIPTION
**Pull request by Abhishek01039**
I have added some Documentation of static method.

### CODE REVIEW

**Comment by Abhishek01039** [comment]
@mit-mit 
can you please review this PR?
Thanks.

**Comment by mit-mit** [comment]
Isn't this part (calling static methods) already covered right above the code snippet? cc @kwalrath

**Comment by Abhishek01039** [comment]
This part is covered by the code snippet.
but, is this not better to have well-documented these things also?
Thanks.

**Comment by mit-mit** [comment]
@kwalrath is a better wordsmith than I, but my point was that I think this belongs in the paragraph above the code snippet. That currently has:

```
Static methods (class methods) do not operate on an instance, and thus do not have access to this. For example:
```

Perhaps it could have:

```
Static methods (class methods) do not operate on an instance, and thus do not have access to this. 
Static method can be invoked directly on a class, and can access can access static data member.
For example:

```

**Comment by Abhishek01039** [comment]
Yeah 
Thanks, @mit-mit

**Comment by kwalrath** [comment]
@leafpetersen could you (or someone) review this for correctness and terminology?

**Review by kwalrath — APPROVED** [review]
I'm going to say this is correct, since it matches what my tests have shown.

**Comment by Abhishek01039** [comment]
Thanks @kwalrath

**Comment by kwalrath** [comment]
> Thanks @kwalrath

Thank _you_ @Abhishek01039!

File: src/_guides/language/language-tour.md [hunk 1]
```diff
@@ -3536,8 +3536,11 @@ Static variables aren’t initialized until they’re used.
 
 #### Static methods
 
-Static methods (class methods) do not operate on an instance, and thus
-do not have access to `this`. For example:
+Static methods (class methods) don't operate on an instance, and thus
+don't have access to `this`.
```
Comment (leafpetersen) [inline src/_guides/language/language-tour.md:3540]: You could also comment on the fact that they can't reference generic parameters to the class (or not).

**Review by leafpetersen — COMMENTED** [review]
LGTM

**Pull request merged**